src/routes/sig.py

```
@@ -136,17 +136,18 @@ async def join_sig(id: int, session: SessionDep, request: Request):
 
 
 @sig_router.post('/sig/{id}/member/leave', status_code=204)
-async def leave_sig(id: int, session: SessionDep, scsc_global_status: SCSCGlobalStatusDep, request: Request):
+async def leave_sig(id: int, session: SessionDep, request: Request):
     current_user = get_user(request)
     sig = session.get(SIG, id)
     if not sig: raise HTTPException(404, detail="해당 id의 시그/피그가 없습니다")
```

Ruff (0.12.2)

Multiple statements on one line (colon)

(E701)